### ISSUE TITLE
ClassNotFoundException: StatusAggregator for Spring Boot < 2.2

### ISSUE THREAD

**Issue by aksh1618** [resilience4j/resilience4j#1435]
Resilience4j version: 1.7.0

Java version: 8

Using `resilience4j-spring-boot2` with spring boot version < 2.2 and actuator results in this exception:
```
Caused by: java.lang.ClassNotFoundException: org.springframework.boot.actuate.health.StatusAggregator
```
due to `CircuitBreakersHealthIndicatorAutoConfiguration` and `RateLimitersHealthIndicatorAutoConfiguration` classes being loaded, which depend on `StatusAggregator` class which was added in Spring boot 2.2.0. This is fixable by simply adding this class to the `ConditionalOnClass` annotation, and we are currently using resilience4j packaged with just that change with spring boot version 2.0.x in our projects. I'll submit a PR for the same.

**Comment by RobWin** [comment]
Thx

**Comment by celikfatih** [comment]
In version 1.7.0 it still looks the same as before. When will the version with the development be released?

**Comment by aksh1618** [comment]
@celikfatih I think it's already released as part of 1.7.1
